<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-18</y>
         <width>719</width>
         <height>1222</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QFrame" name="frame">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>1200</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
border: 1px solid black;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QScrollArea" name="scrollArea_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>460</y>
             <width>701</width>
             <height>361</height>
            </rect>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>1022</width>
              <height>338</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QFrame" name="frame_2">
               <property name="minimumSize">
                <size>
                 <width>1000</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 158, 158);</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <widget class="QLabel" name="label_4">
                <property name="geometry">
                 <rect>
                  <x>50</x>
                  <y>10</y>
                  <width>181</width>
                  <height>261</height>
                 </rect>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap>../Image/HomeAlone phim.jpg</pixmap>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
               <widget class="QPushButton" name="pb_watch1">
                <property name="geometry">
                 <rect>
                  <x>90</x>
                  <y>280</y>
                  <width>93</width>
                  <height>28</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
                </property>
                <property name="text">
                 <string>Xem</string>
                </property>
               </widget>
               <widget class="QLabel" name="label_5">
                <property name="geometry">
                 <rect>
                  <x>290</x>
                  <y>10</y>
                  <width>181</width>
                  <height>261</height>
                 </rect>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap>../Image/It's a Wonderful Life phim.jpg</pixmap>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
               <widget class="QPushButton" name="pb_watch2">
                <property name="geometry">
                 <rect>
                  <x>330</x>
                  <y>280</y>
                  <width>93</width>
                  <height>28</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
                </property>
                <property name="text">
                 <string>Xem</string>
                </property>
               </widget>
               <widget class="QLabel" name="label_6">
                <property name="geometry">
                 <rect>
                  <x>530</x>
                  <y>10</y>
                  <width>181</width>
                  <height>261</height>
                 </rect>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap>../Image/A Christmas Story phim.jpg</pixmap>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
               <widget class="QPushButton" name="pb_watch3">
                <property name="geometry">
                 <rect>
                  <x>570</x>
                  <y>280</y>
                  <width>93</width>
                  <height>28</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
                </property>
                <property name="text">
                 <string>Xem</string>
                </property>
               </widget>
               <widget class="QLabel" name="label_7">
                <property name="geometry">
                 <rect>
                  <x>790</x>
                  <y>10</y>
                  <width>181</width>
                  <height>261</height>
                 </rect>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap>../Image/The Nightmare Before Christmas phim.jpg</pixmap>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
               <widget class="QPushButton" name="pb_watch4">
                <property name="geometry">
                 <rect>
                  <x>840</x>
                  <y>280</y>
                  <width>93</width>
                  <height>28</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
                </property>
                <property name="text">
                 <string>Xem</string>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QLabel" name="la_sanpham3">
           <property name="geometry">
            <rect>
             <x>480</x>
             <y>140</y>
             <width>221</width>
             <height>141</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../Image/images (1).jpg</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="lb_sanpham1">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>290</y>
             <width>231</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 0px;
font: 9pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Giấy gửi thư Noel cho bạn bè Noel</string>
           </property>
          </widget>
          <widget class="QLabel" name="la_sanpham1">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>140</y>
             <width>221</width>
             <height>141</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../Image/ban-thiep-3d-giang-sinh-noel-tayta4.jpg</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="lb_sanpham3">
           <property name="geometry">
            <rect>
             <x>480</x>
             <y>290</y>
             <width>221</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 0px;
font: 9pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Thiếp giáng sinh ông già noel đẹp </string>
           </property>
          </widget>
          <widget class="QLabel" name="la_sanpham2">
           <property name="geometry">
            <rect>
             <x>250</x>
             <y>140</y>
             <width>221</width>
             <height>141</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../Image/thiep-3d-cay-thong-noel-dep-moi-nhat5-8318.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="lb_sanpham2">
           <property name="geometry">
            <rect>
             <x>280</x>
             <y>290</y>
             <width>161</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 0px;
font: 9pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Thiệp giáng sinh tuần lộc</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pb_buy1">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>320</y>
             <width>93</width>
             <height>28</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Xem</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pb_buy2">
           <property name="geometry">
            <rect>
             <x>320</x>
             <y>320</y>
             <width>93</width>
             <height>28</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Xem</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pb_buy3">
           <property name="geometry">
            <rect>
             <x>550</x>
             <y>320</y>
             <width>93</width>
             <height>28</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Xem</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>1070</y>
             <width>341</width>
             <height>71</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 0px;
font: 11pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Top những món quà mà bạn cần tặng cho người yêu của bạn.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="la_sanpham4_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>860</y>
             <width>331</width>
             <height>221</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../Image/Món quà nên(Bài viết).jpg</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="pb_watch5">
           <property name="geometry">
            <rect>
             <x>250</x>
             <y>1130</y>
             <width>93</width>
             <height>28</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Xem</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>1070</y>
             <width>341</width>
             <height>71</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 0px;
font: 11pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Tại đường phố NewYork,mọi người tập trung đông đúc để đón giáng sinh.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="pb_watch6">
           <property name="geometry">
            <rect>
             <x>620</x>
             <y>1130</y>
             <width>93</width>
             <height>28</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Xem</string>
           </property>
          </widget>
          <widget class="QLabel" name="la_sanpham4_3">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>860</y>
             <width>331</width>
             <height>221</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../Image/Đường phố New York.jpg</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>230</x>
             <y>390</y>
             <width>271</width>
             <height>51</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Bảng xếp hàng bộ những bộ phim Giáng sinh hay nhất.</string>
           </property>
           <property name="textFormat">
            <enum>Qt::PlainText</enum>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QWidget" name="widget_2" native="true">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>10</y>
             <width>551</width>
             <height>111</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius: 20px;
background-color: rgb(255, 92, 92);</string>
           </property>
           <widget class="QPushButton" name="pushButton_2">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>101</width>
              <height>81</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../Image/Logo App.png</normaloff>../Image/Logo App.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
           </widget>
           <widget class="QPushButton" name="pb_menu_2">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>40</y>
              <width>131</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>Menu</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pb_contact">
            <property name="geometry">
             <rect>
              <x>280</x>
              <y>40</y>
              <width>121</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>Contact</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pb_logout">
            <property name="geometry">
             <rect>
              <x>410</x>
              <y>40</y>
              <width>121</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>Logout</string>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>764</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
